apps/coordinator/src/components/Wallet/TransactionPreview.jsx

```diff
@@ -165,6 +166,20 @@ class TransactionPreview extends React.Component {
       unsignedPSBT,
     } = this.props;
 
+    const wasteMetrics = new WasteMetrics();
+    const weight = fee / satoshisToBitcoins(feeRate);
+    // Doubt-1: How do I know what is the amount that I want to actually spend and what is change amount?
+    const spendAmount = 10000;
+    // Doubt-2: In caravan context, How do we want to plan L value?
+    const L = 100; // longTermFeeEstimate
```

Comment:
For now, Just to try out the functionality, I have hardcoded random values. But how would we populate here the actual values ?

Reply:
For the first doubt- all of that information is there on the transaction preview screen. You can see the change output has a little `(change)` marked there. Since this is a transaction that is created by the user, change is known because it has to be decided by the client/application ahead of time. 

"amount to spend" are the other outputs

<img width="1278" alt="Screenshot 2024-09-07 at 8 31 27 PM" src="https://github.com/user-attachments/assets/b7d1112e-aa02-4543-89aa-1813f1cd7105">

For `L` this is a good question. You could come up with some formula to estimate and hardcode with explanation. If you're going to do that, then I'd suggest having that in the package to start with and using that as a default. 

I think an overall better UX would be to default to the current "fast" fee rate (maybe with some percentage above as a buffer) and then in addition have a slider (something like [this](https://mui.com/material-ui/react-slider/) to use as the component. an example of a similar implementation is [Live Wallet](https://github.com/Jwyman328/LiveWallet)) that allows the user to set it themselves and see how that impacts the rating of their transaction.